session_db/readme/ROADMAP.md

```diff
@@ -0,0 +1,2 @@
+This module does not work with multi-threaded workers, so it requires
+workers \> 0.
```

Comment:
Isn't this implemented now? Since https://github.com/OCA/server-tools/commit/8023bff1b654ab0d27a6aebcdee4f7edb9dd3d0b

Reply:
I removed ROADMAP.md